templates/CRM/Civicase/Admin/Form/Settings.tpl

```diff
@@ -13,7 +13,7 @@
 <tr class="crm-case-form-block-webformUrl">
   <td class="label">{$form.civicaseWebformUrl.label}</td>
   <td>{$form.civicaseWebformUrl.html}<br />
-    <span class="description">{ts}A Webform url e.g node/233{/ts}</span>
+    <span class="description">{ts}A Webform url e.g /node/233{/ts}</span>
```

Comment:
Are we sure we need to add a forward slash before the webform URL? tbh, not sure myself, just double checking with you.

Reply:
Yes @tunbola, this is what happens in different scenarios taking into consideration we currently are in `http://civicase.local/civicrm/case/a/#/case` (case dashboard):

* if the webform url value is `node/233` it will try to go to `http://civicase.local/civicrm/case/node/233`.
* if the webform url value is `/node/233` it will go to `http://civicase.local/node/233`.
* if the url value is `http://example.com` it will go to `http://example.com`.

So if we just add `node/233` it will go to a place that does not even exist because it will try to solve a relative path.